setup.py

```diff
@@ -1,9 +1,38 @@
 #!/usr/bin/env python
 """setup.py for shorah."""
+import glob
 import sys
+import os
+import shutil
 
 from setuptools import setup, find_packages
 from setuptools.command.test import test as TestCommand
+from setuptools.command.develop import develop
+from setuptools.command.install import install
+
+
+def move_files():
+    """Identify the build directory and copy executables into src/shorah."""
+    try:
+        diri_file = glob.glob('*/src/cpp/diri_sampler')[0]
```

Comment:
This mechanism will successfully work only when there's a single build directory.
(e.g.: only when making a single directoy called `build`)

It can pick the wrong directory if there are multiple builds.
(e.g.: IDEs like KDevelop that can handle separate "debug" and "optimized" builds in different directories).

Reply:
Is there a reason for us to have multiple build dirs when installing with `pip`? By the way, in that case it would not exactly fail, but just copy the executables from the first directory found. Am I right?

Reply:
Yes, I was thinking about the situation where an IDE creates `build-debug/` in addition to `build`, and pip picks up from the other directory.